entrypoint.sh

```diff
@@ -20,10 +20,16 @@ set -e
 export KS_CLIENT="github_actions"
 
 if [ -n "${INPUT_FRAMEWORKS}" ] && [ -n "${INPUT_CONTROLS}" ]; then
-  echo "Framework and Control are specified. Please specify either one of them or neither"
+  echo "Framework and Control are specified. Please specify either one of them"
```

Comment:
Nit remove either

Reply:
I have changed the script so the user *must* define either one of them, but if neither the action fails. 

If we define neither, as of Kubescape 3.0, we scan the "cluster security framework" (the command which is executed is `kubescape scan` without framework or control) and it has no meaning in the use case of the GH action